misc/dist/html/service-worker.js

```diff
@@ -161,7 +161,7 @@ self.addEventListener('message', (event) => {
 		} else if (msg === 'update') {
 			self.skipWaiting().then(() => self.clients.claim()).then(() => self.clients.matchAll()).then((all) => all.forEach((c) => c.navigate(c.url)));
 		} else {
-			onClientMessage(event);
+			onClientMessage(event); // eslint-disable-line no-undef -- bug? https://github.com/godotengine/godot/pull/91771#issuecomment-2104999411
```

Comment:
We should just remove this line along with the whole else block.

Reply:
ok